### ISSUE TITLE
Docs gone missing

### ISSUE THREAD

**Issue by Cruiser13** [dachcom-digital/pimcore-dynamic-search#14]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no

Looks like the docs of this repository are gone somehow. Everything below Example Setup is gone.
The following docs are expected from Readme.md

Configuration
    Context Guard
    Document Definition
    Logging
Data Creation
    Enable automatic Update / Insert / Delete Service
Data Fetching
    Output Channels
        Create Output Channel
        Multi Search Channels
        Channel Filter / Actions
    Filter (Faceted Search / Aggregation)
        Create Filter Definition
API

**Comment by dpfaffenbauer** [comment]
they never existed, there are no docs yet

**Comment by Cruiser13** [comment]
Are you sure? I'm sure I read them some time ago.

**Comment by dpfaffenbauer** [comment]
I am very certain. only thing there is is this:

https://github.com/dachcom-digital/pimcore-dynamic-search/blob/master/docs/0_ExampleSetup.md

**Comment by Cruiser13** [comment]
You are right. Mixed it up with the  dachcom-digital / pimcore-lucene-search repository.
There are the docs I had in mind. I'll turn this into a feature request in this case ;-)

**Comment by solverat** [comment]
Never existed. There is currently no time for the most important thing. :) But we're working on it!

**Comment by dpfahlbusch** [comment]
Are there any new informations about the current progress? :)

**Comment by dpfaffenbauer** [comment]
@dpfahlbusch nothing changed so far...

**Comment by DzemalBecirevic** [comment]
Hi,
I'm also missing documentation. I'm trying to get dynamic search working with WebCrawler and Elastic Search and can not find any documentation on how to implement this.

Are there any plans on updating documentation with this, or is there any example you can provide?

**Comment by Cruiser13** [comment]
@solverat we're missing the same thing as @DzemalBecirevic . Any chance you can give us a hint here? I'd be happy to provide a full documentation pr for this case if we can get this to work.

**Comment by openconcept** [comment]
It would go a long way with just a complete sample configuration/setup for Trinity and Elasticsearch with document/object/asset indexing and search.

**Comment by aarongerig** [comment]
Meanwhile we have [another setup example](https://github.com/dachcom-digital/pimcore-dynamic-search/blob/master/docs/02_MultiSiteMultiLocaleSetup.md) of a more complex use case (multi-locale, multi-site).
Hope this helps out a bit!
